File: course_code/std_io/std_io.c
#include <stdio.h>
#include <stdlib.h>

int main(){
  int test_cases = 0;
  int arg1, arg2;
  scanf("%d", &test_cases);
  while (test_cases--){
    scanf("%d", &arg1);
    scanf("%d", &arg2);
    // perform operation here
    printf("%d\n",(arg1 + arg2));
  }
}

File: course_code/std_io/std_io.py
test_cases = raw_input() #Default input type is string
test_cases = int(test_cases)
while test_cases:
	arg1,arg2  = raw_input().split()
	arg1 = int(arg1)
	arg2 = int(arg2)
	print arg1 + arg2
	test_cases = test_cases - 1